textBasedAdventure.js

```diff
@@ -1,5 +1,53 @@
-const readline = require('readline-sync')
+const readline = require("readline-sync")
 
-let nameInput = readline.question("Enter your name: ")
+const play = () => {
+    userInput = readline.question("This life I tell ya, want me to tell you about it; yes, or no?")
+    while(userInput !== "yes") {
+        console.log("Oh, I'm sure you have the time.")
+    }
+}
+console.log("Thanks, I'd be glad to tell you. Anyone can have a rough time, but let me tell you; with my life, lady luck doesn't just ignore me she sent me a restraining order.")
 
-console.log(`Hello ${nameInput}!  Welcome to my game.`)
+const dice = () => {
+    userInput =  readline.question("I tell ya, me and my wife were happy for twenty years. Would you believe that? Yes / No")
+    while(userInput !== "Yes") {
+        console.log("I'm waiting.")
+    }
+}
+console.log("Yeah then we met. But that's not totally true, even as a kid I got the short end of the stick. I asked my dad if I counld go out ice skating at the local pond, and he told me, wait till it gets warmer.")
+
+const noDice = () => {
+    userInput = readline.question("It doesn't just go for people either. You get where I'm coming from? Yes / No")
+    while(userInput !== "No"){
+        console.log("I doubt it.")
+    }
+}
+const aLikelyStory = () => {
+   userInput = readline.question("Cause the thing is people say that eventually people and their pets start to look alike. My dog found out about that and he ran away. Let me tell you another story. Yes / No")
+    if(userInput === "yes"){
+        console.log("I had no idea you cared so much")
```

Comment:
You can add other functions inside the player choices to keep the flow of the game going.
So you can add the response to their choice, then execute a function that leads them to another choice.

Reply:
you mean in the if statement or pass in their response to the readline question as a parameter on the next function?

Reply:
ah never mind I just need to invoke another function before the final closing brackets